container/dockerhub/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y software-properties-common
```

Comment:
I think this was required for adding the other repository for clang-format 9 and should be removed as well.

Reply:
ok